docs/source/reference/tpu.rst

```diff
@@ -1,25 +1,30 @@
 .. _tpu:
 
+=========
 Cloud TPU
-================================
+=========
+
+Overview
+========
 
 SkyPilot supports running jobs on Google's `Cloud TPU <https://cloud.google.com/tpu/docs/intro-to-tpu>`_.
 Two different TPU architectures are available on GCP:
 
-- `TPU Nodes <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-node>`_
 - `TPU VMs <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-vm>`_
+- `TPU Nodes <https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu-node>`_
 
-Both are supported by SkyPilot.
+Both are supported by SkyPilot. We recommend TPU VMs which is a newer architecture encouraged by GCP.
 
 The two architectures differ as follows.
-For TPU Nodes, a host VM communicates with the TPU host over gRPC.
-For TPU VMs, you can SSH directly into a VM that is physically connected to the TPU device.
+For TPU VMs, you can directly SSH into the "TPU host" VM that is physically connected to the TPU device.
+For TPU Nodes, a user VM (e.g., `n1` instances) must be separately provisioned to communicate with an inaccessible TPU host over gRPC.
```

Comment:
```suggestion
For TPU Nodes, a user VM (an ``n1`` instance) must be separately provisioned to communicate with an inaccessible TPU host over gRPC.
```

Reply:
I used e.g. because it can be other type of instance rather than `n1`. do you think an is better?

Reply:
Hmm, I was under the impression that
```
» sky tpunode -t n2d-standard-8
sky.exceptions.ResourcesMismatchError: TPU Nodes can be only used with N1 machines. Please refer to: https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines
```

Reply:
Ah okay I didn't know this. @WoosukKwon Seems the check was added by you. Is this the case?